benchmark/poseidon/simple_poseidon_benchmark_reporter.cc

```diff
@@ -0,0 +1,38 @@
+#include "benchmark/poseidon/simple_poseidon_benchmark_reporter.h"
+
+#include <stddef.h>
+
+#include <string>
+#include <vector>
```

Comment:
Could you remove this?  This header will be included in the header that contains a parent class